app/pods/publications/publication/template.hbs

```diff
@@ -117,13 +117,11 @@
                                               @selected={{this.getPublicationType}}
                 />
               </div>
-              <div class="auk-form-group">
-                {{web-components/vl-checkbox
-                  value=(gt this.model.publicationFlow.priority 0)
-                  label=(t "priority-procedure")
-                  toggle=(action "togglePriorityProcedure")
-                }}
-              </div>
+
+              <Publications::UrgencyLevelCheckbox
+                      @urgencyLevel={{ this.model.publicationFlow.urgencyLevel }}
```

Comment:
Deze relatie is nog niet geladen, dus je stuurt hier een promise naar je component.
Moet hier die getter gebruikt worden ?